protocol.html

```diff
@@ -267,7 +267,12 @@ <h2>Editor’s Draft, 2022-01-31</h2>
           <section id="abstract">
             <h2>Abstract</h2>
             <div datatype="rdf:HTML" property="schema:abstract">
-              <p>Solid Notifications Protocol.</p>
+              <p>
+                This specification defines a linked data-based protocol for subscribing to resources
```

Comment:
Intend to be DesignIssues' LD or using it more liberally?

```suggestion
                This specification defines a Linked Data-based protocol for subscribing to resources
```